### PR TITLE
feat: bundle analyzer endpoint added

### PR DESCRIPTION
**Pull request by lpk3nny**

### CODE REVIEW

File: server/routes/__analyze.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+				omit = [], hostname = 'polyfill.qiwi.com', unknown,
+				useComputeAtEdgeBackend, flags,
+			},
+		},
+		response,
+	) => {
+		const bundleFileNames = files.map(({filename}) => filename)
+
+		const result = await bundleAnalyzer({
+			file: bundleFileNames, cwd: `./${uploadDestination}`,
```
Comment (antongolub) [inline server/routes/__analyze.js]: let's use `tempy.temporaryDirectory()`

**Review by antongolub — CHANGES_REQUESTED** [review]

File: server/utils/config.js [hunk 2]
```diff
@@ -9,14 +9,16 @@ module.exports = {
...
 
 	get CORSAllowedFirstLevelDomains() {
-		return process.env.ALLOWED_DOMAINS
-			? new Set(process.env.ALLOWED_DOMAINS.split(','))
-			: new Set();
+		return process.env.ALLOWED_DOMAINS ? process.env.ALLOWED_DOMAINS.split(',') : [];
 	},
 
 	get serveStaticSite() {
 		if (process.env.SERVE_STATIC_SITE) return Boolean(process.env.SERVE_STATIC_SITE);
```
Comment (antongolub) [inline server/utils/config.js]: Boolean('false') → true. Boolean('0') → true.
Let's add util parseBool helper:
```js
parseBool = (value) => value === 'true' || value === '1' || value === 1 || value === true || false
```

**Review by antongolub — CHANGES_REQUESTED** [review]

**Review by antongolub — COMMENTED** [review]
Almost LGTM)

File: server/utils/config.js (hunk 2 above)
Comment (lpk3nny) [reply server/utils/config.js]: ri-ight

**Review by antongolub — APPROVED** [review]

**Pull request merged**